src/components/myEvents/MyEvents.tsx

```diff
@@ -130,10 +134,10 @@ const EventRow = ({ event }: EventRowProps) => {
               )}
               {hostNames && (
                 <div className="flex items-center gap-2">
-                  <Crown size={16} />
+                  <Crown size={16} className={isHost ? "text-primary" : ""} />
 
                   <div>
-                    <p className="w-72 overflow-hidden text-ellipsis text-nowrap">
+                    <p className={"w-72 overflow-hidden text-ellipsis text-nowrap" + (isHost ? " text-primary font-bold" : "")}>
```

Comment:
Great idea, but I think this looks a bit too intense. How about we use text-primary on the crown, and text-primary + underline on the hostnames. And then skip the bold. I think that will be clear while not messing as much with the visual priority of the different font hierarchies.